user-guide/Reference/DataMiner_Extension_Modules.md

```diff
@@ -20,6 +20,12 @@ For ease of installation, DxMs may be packaged together in a pack, such as the [
 
 The APIGateway module is the endpoint for the gRPC communication with a DataMiner Agent.
 
+From DataMiner 10.4.2/10.5.0 onwards<!--RN 37734 + 36575-->, DataMiner modules can register with APIGateway. All modules registered with APIGateway are displayed under `/APIGateway/api/version`, showing the following properties:
+
+- Name
+- Version
```

Comment:
@BertVandenberghe I recall you mentioning something about the security aspect of exposing version information.
Should the version be left out of the documentation?

Reply:
No, the version can be left in. The documentation can indicate in which version which features have been added. That's not a concern. An API should not expose the version via an unauthenticated method because that would be considered 'information leakage'. This can be used to query if systems are running outdated versions which could maybe contain vulnerabilities. But documentation can mention which version now includes a new feature, no problem.